src/app/core/component.ts

```diff
@@ -55,9 +55,9 @@ export default class Component {
 
         // Create and set the ports here itself
 
-        const cleanparamdata = this._params.parameters;
+        const cleanparamdata = this._params.toMap();
```

Comment:
@MurrayLF If we aren't gonna access parameters anymore, we should make it private. It's been one of the bigger issues in this Parameters API

Reply:
It is still being used in a couple places